themes/westerndevs/scripts/author-list-json.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+hexo.extend.generator.register('author-list-json', function(locals) {
+
+  var authorList = [];
+  var authorKeys = Object.keys(locals.data.authors);
+  for (var i = 0; i < authorKeys.length; i++) {
+    var authorId = authorKeys[i];
+    var author = locals.data.authors[authorId];
```

Comment:
I would love to be able to use `filter` on `locals.data.authors` but it's not an array. Even if we could, the yaml is structured such that the object doesn't include the author's ID. Open to suggestions because I've run into this a number of times.